src/lib/fetch.ts

```diff
@@ -38,25 +38,35 @@ export {
   setMinaGraphqlFallbackEndpoints,
   setArchiveGraphqlEndpoint,
   setArchiveGraphqlFallbackEndpoints,
+  setAccountsManagerEndpoint,
   sendZkappQuery,
   sendZkapp,
   removeJsonQuotes,
   fetchEvents,
   fetchActions,
+  KeyPair,
+  acquireKeyPair,
+  releaseKeyPair
 };
 
 type NetworkConfig = {
   minaEndpoint: string;
   minaFallbackEndpoints: string[];
   archiveEndpoint: string;
   archiveFallbackEndpoints: string[];
+  accountsManagerEndpoint: string;
+};
+type KeyPair = {
+  publicKey: PublicKey;
+  privateKey: PrivateKey;
```

Comment:
can you remove the `KeyPair` type and just use an inline type for the function which returns a keypair? I think we shouldn't export more things than necessary.